client/i18n/locales/english/translations.json

```diff
@@ -517,7 +517,7 @@
     "opens-new-window": "Opens in new window"
   },
   "flash": {
-    "honest-first": "To claim a certification, you must first agree to our academic honesty policy",
+    "honest-first": "To claim a certification, you must first accept our academic honesty policy",
```

Comment:
```suggestion
    "honest-first": "To claim a certification, you must first agree to our academic honesty policy",
```

Reply:
This change is out of scope. Please revert the changes or commit the suggestion.